internal/clients/interface.go

```diff
@@ -89,6 +90,12 @@ var GetServiceAzureClient func(ctx context.Context) (ServiceAzure, error)
 
 type Azure interface {
 	ClientStatuser
+
+	// CreateVM creates Azure virtual machine.
+	// Most of the parameters are constant for now.
+	// imageID is expected in format /subscriptions/{subscriptionID}/resourceGroups/{resourceGroupName}/providers/Microsoft.Compute/images/{imageID}
+	// sshKeyBody should be full public key body
+	CreateVM(ctx context.Context, imageID string, pubkey *models.Pubkey, instanceType InstanceTypeName) (*armcompute.VirtualMachine, error)
```

Comment:
One important thing - I would like us not to cross the API boundary, can you return just instance IDs like we do in AWS? If we need more, new type must be introduced.